api/utils/requestProcessor.js

```diff
@@ -537,27 +537,21 @@ const processRequest = (params) => {
 
                 switch (paths[3]) {
                 case 'create':
-                    validateUserForWriteAPI(() => {
-                        if (!(params.member.global_admin)) {
-                            common.returnMessage(params, 401, 'User is not a global administrator');
-                            return false;
-                        }
-                        countlyApi.mgmt.apps.createApp(params);
-                    }, params);
+                    validateUserForGlobalAdmin(params, countlyApi.mgmt.apps.createApp);
```

Comment:
This would not be Create for "global_applications" too?

Reply:
yes, it should be and changed already. :)